src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -500,22 +500,38 @@ public override Size ClientSize
             }
         }
 
-        public override bool CursorVisible
+        public override bool CursorGrabbed
```

Comment:
Can you elaborate a bit on this change?

Reply:
don't let yourself be mislead by git diff, It seems to me that CursorVisible is still the same, but CursorGrabbed got added.

Reply:
These changes are part of PR #589 which I cherry picked into my fork. If I remember correctly it was necessary to fix some issues an mac os as well as on linux, but @copygirl should know more about the linux part.